app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -83,7 +83,7 @@ def invoice
     template = if Spree::Config.invoice_style2? then "spree/admin/orders/invoice2" else "spree/admin/orders/invoice" end
     pdf = render_to_string pdf: "invoice-#{@order.number}.pdf", template: template, formats: [:html], encoding: "UTF-8"
     Spree::OrderMailer.invoice_email(@order.id, pdf).deliver
-    flash[:success] = t(:invoice_email_sent)
+    flash[:success] = t('admin.orders.invoice_email_sent')
```

Comment:
@sauloperez I'm happy to close this branch if you've got both the keys added elsewhere.